spec/std/string_spec.cr

```diff
@@ -229,6 +229,18 @@ describe "String" do
     it "gets byte_slice with negative index" do
       "hello".byte_slice(-2, 3).should eq("lo")
     end
+
+    it "gets byte_slice(Int) with with start out of bounds" do
```

Comment:
There's a duplicate `with`.
And these two specs could be combined into one, they test the same thing (identical description).